Library/Homebrew/cmd/test-bot.rb

```diff
@@ -244,22 +241,9 @@ def safe_formulary(formula)
     end
 
     def git(*args)
-      rd, wr = IO.pipe
-
-      pid = fork do
-        rd.close
-        STDERR.reopen("/dev/null")
-        STDOUT.reopen(wr)
-        wr.close
-        Dir.chdir @repository
-        exec("git", *args)
+      Dir.chdir(@repository) do
+        Utils.popen_read("git", *args)
       end
```

Comment:
That would apply to a lot of places in Homebrew, by the way. Note that `-C <path>` was “only” added in Git 1.8.5, so the answer might depend on what minimum Git version we want to assume (or enforce, by checking for it). For reference, Git 1.8.5 first appeared on OS X with Xcode 5.1.